tictactoe.Tests/Game.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.IO;
+using Xunit;
+using tictactoe;
+
+namespace tictactoe.Tests
+{
+    public class GameTest
+    {
+       [Fact]
+       public void MenuTest()
+        {
+            var stringWriter = new StringWriter();
+            Console.SetOut(stringWriter);
+
+            Game game = new Game();
+            var expected = "Welcome to TicTacToe!Game Options:1. Human v. Human";
+
+            game.Menu();
+
+            var actual = stringWriter.ToString().Replace(Environment.NewLine, "");
```

Comment:
Minor point but instead of replacing new line with empty string here, you can also insert the new line character in your expected string.

Reply:
commit 3f991a54ed932e14aceabbd444f9a4711323f5ff